

# BioHackathon Europe projects 2024
This repository is intended for the BioHackathon Europe participants to share ideas. The event will take place in Campus Belloch, 4-8 November 2024. For more information, please see the [BioHackathon Europe website](https://biohackathon-europe.org/index.html).

## Accepted projects

* [1. Accelerate your bioinformatics workflows with CGG AI Cloud](1.md)
* [2. A curated assessment of metadata descriptors of AI-ready datasets](2.md)
* [3: Reusable RDM Planning Environments for Trainings and Workshops](3.md)
* [4. SPARQL Query Generation for Efficient Scientific Data Access of ELIXIR resources](4.md)
* [5. Mapping of research software quality indicators across the ELIXIR Research Software Ecosystem](5.md)
* [6. Gender representation in ELIXIR-supported publications: a visibility analysis across academic search engines](6.md)
* [7. Enhancing the Training metrics database (TMD) for improved reporting](7.md)
* [8. Data Model Converter: Bridging Cohort Information Across Biomedical Data Models](8.md)
* [9. BioHackrXiv: improving biohackathon publications](9.md)
* [10. BioSchemas for mortals](10.md)
* [11. Galaxy CoDex - Ensuring Galaxy community sustainability through resource aggregation and annotation](11.md)
* [12. Perturb -Bench: large-scale benchmarking of perturbational modelling tools in complex single-cell data](12.md)
* [13. Interconnecting identifiers.org into a broader metadata connectivity](13.md)
* [14. FAIRly easy APIs for research data in (Bio)Schema.org and RDF](14.md)
* [15. Enhancing interoperability of biomedical resources using ontologies.](15.md)
* [16. Enhancing bio.tools by Semantic Literature Mining](16.md)
* [17. Development of FAIR image analysis workflows and training in Galaxy](17.md)
* [18. Expanding FAIR database integration through elucidation and transformation of underlying graph schemas.](18.md)
* [19. Creating user benefit from ARC-ISA RO-Crate machine-actionability](19.md)
* [20. Structuring Clinical Reports into OMOP Common Data Model (CDM)](20.md)
* [21. Enhancing multi-omic analyses through a federated microbiome analysis service](21.md)
* [22. Enabling Secure Data Access from Galaxy to (F)EGA](22.md)
* [23. MARS: Multi-omics Adapter for Repository Submissions, preparing for launch](23.md)
* [24. Increasing FAIRness of digital agrosystem resources by extending Bioschemas](24.md)
* [25. Recognising research software contributions leveraging the ELIXIR infrastructure](25.md)
* [26. Reducing the environmental impact of Galaxy](26.md)
* [27. Integrating Bioconductor packages with the ELIXIR Research Software Ecosystem using EDAM](27.md)
* [28. Semantic Interoperability of Biomedical Data](28.md)
* [29. ELIXIR FAIR Lesson Plan Handbook: advancing researchers’ & data stewards’ FAIR skills](29.md)
* [30. The BioHackathon Cloud](30.md)
* [31. Executable metadata mappings to FAIRify Biodiversity Genome Annotations](31.md)
* [32. VCF Explorer: Empowering Genomic Data Interaction](32.md)

<ul>
<ol><a href="1.md">1. Accelerate your bioinformatics workflows with CGG AI Cloud</a></ol>
<ol><a href="2.md">2. A curated assessment of metadata descriptors of AI-ready datasets</a></ol>
<ol><a href="3.md">3: Reusable RDM Planning Environments for Trainings and Workshops</a></ol>
<ol><a href="4.md">4. SPARQL Query Generation for Efficient Scientific Data Access of ELIXIR resources</a></ol>
<ol><a href="5.md">5. Mapping of research software quality indicators across the ELIXIR Research Software Ecosystem</a></ol>
<ol><a href="6.md">6. Gender representation in ELIXIR-supported publications: a visibility analysis across academic search engines</a></ol>
<ol><a href="7.md">7. Enhancing the Training metrics database (TMD) for improved reporting</a></ol>
<ol><a href="8.md">8. Data Model Converter: Bridging Cohort Information Across Biomedical Data Models</a></ol>
<ol><a href="9.md">9. BioHackrXiv: improving biohackathon publications</a></ol>
<ol><a href="10.md">10. BioSchemas for mortals</a></ol>
<ol><a href="11.md">11. Galaxy CoDex - Ensuring Galaxy community sustainability through resource aggregation and annotation</a></ol>
<ol><a href="12.md">12. Perturb -Bench: large-scale benchmarking of perturbational modelling tools in complex single-cell data</a></ol>
<ol><a href="13.md">13. Interconnecting identifiers.org into a broader metadata connectivity</a></ol>
<ol><a href="14.md">14. FAIRly easy APIs for research data in (Bio)Schema.org and RDF</a></ol>
<ol><a href="15.md">15. Enhancing interoperability of biomedical resources using ontologies.</a></ol>
<ol><a href="16.md">16. Enhancing bio.tools by Semantic Literature Mining</a></ol>
<ol><a href="17.md">17. Development of FAIR image analysis workflows and training in Galaxy</a></ol>
<ol><a href="18.md">18. Expanding FAIR database integration through elucidation and transformation of underlying graph schemas.</a></ol>
<ol><a href="19.md">19. Creating user benefit from ARC-ISA RO-Crate machine-actionability</a></ol>
<ol><a href="20.md">20. Structuring Clinical Reports into OMOP Common Data Model (CDM)</a></ol>
<ol><a href="21.md">21. Enhancing multi-omic analyses through a federated microbiome analysis service</a></ol>
<ol><a href="22.md">22. Enabling Secure Data Access from Galaxy to (F)EGA</a></ol>
<ol><a href="23.md">23. MARS: Multi-omics Adapter for Repository Submissions, preparing for launch</a></ol>
<ol><a href="24.md">24. Increasing FAIRness of digital agrosystem resources by extending Bioschemas</a></ol>
<ol><a href="25.md">25. Recognising research software contributions leveraging the ELIXIR infrastructure</a></ol>
<ol><a href="26.md">26. Reducing the environmental impact of Galaxy</a></ol>
<ol><a href="27.md">27. Integrating Bioconductor packages with the ELIXIR Research Software Ecosystem using EDAM</a></ol>
<ol><a href="28.md">28. Semantic Interoperability of Biomedical Data</a></ol>
<ol><a href="29.md">29. ELIXIR FAIR Lesson Plan Handbook: advancing researchers’ & data stewards’ FAIR skills</a></ol>
<ol><a href="30.md">30. The BioHackathon Cloud</a></ol>
<ol><a href="31.md">31. Executable metadata mappings to FAIRify Biodiversity Genome Annotations</a></ol>
<ol><a href="32.md">32. VCF Explorer: Empowering Genomic Data Interaction</a></ol>
</ul>
